samples/createTopicWithSchema.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019-2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This sample demonstrates how to perform basic operations on topics with
+ * the Google Cloud Pub/Sub API.
+ *
+ * For more information, see the README.md under /pubsub and the documentation
+ * at https://cloud.google.com/pubsub/docs.
+ */
+
+'use strict';
+
+// sample-metadata:
+//   title: Create Topic With Schema
+//   description: Creates a new topic, with a schema definition.
+//   usage: node createTopicWithSchema.js <topic-name>
+
+async function main(topicName = 'YOUR_TOPIC_NAME') {
+  // [START pubsub_create_topic]
```

Comment:
Is this sample supposed to create a topic with schema? If so, need to pass schema name and update the region tag:

```suggestion
  // [START pubsub_create_topic_with_schema]
```

Reply:
I actually hadn't even started on this one, but your change is right. Thanks :)